pkg/tfbridge/x/tokens.go

```diff
@@ -713,27 +747,24 @@ func aliasResource(
 		hist[tfToken].Fields = nil
 	}
 
-	safeRange(res, func(k string, v shim.Schema) {
-		walk(
-			getNonNil(&hist[tfToken].Fields, k),
-			getNonNil(&computed.Fields, k),
-			v,
-		)
-	})
+	walkResource(res, &hist[tfToken].Fields, &computed.Fields)
 }
 
-func getNonNil[K comparable, V any](m *map[K]*V, key K) *V {
+func getNonNil[K comparable, V any](m *map[K]*V, key K) (_ *V, alreadyThere bool) {
```

Comment:
I'm not familiar with this Go pattern, and I could not find any quick reference to it - why are we returning a blank identifier here?

Reply:
The feature I'm using is called [Named return values
](https://go.dev/tour/basics/7). I wanted to name the second returned value, since I often get these types of bool indicators confused. I didn't feel the need to name the first returned value (type `*V`). `_` is a special identifier in Go that can never be referenced.[^1]

[^1]: If you are familiar with lisp, you can conceptually replace each use of `_` with `(gensym)`.